cssom/interfaces.html

```diff
@@ -29,61 +29,43 @@
 
 <script>
 "use strict";
-var style_element, svg_element, xmlss_pi;
+var style_element = document.getElementById('styleElement');
+var svg_element = document.getElementById('svgElement');
+var xmlss_pi = document.getElementById('xmlssPiIframe').contentDocument.firstChild;
```

Comment:
There's no guarantee that the content document has finished loading yet, right? Shouldn't this be in the load handler?